From: Luca Boccassi <luca.boccassi@gmail.com>
Date: Sat, 14 May 2016 21:44:00 +0100
Subject: Ignore failure to set DRM interface

This always fail due to problems in libdrm, but does not affect
Bumblebee's functionality. See GH-652 for more details.
---
 src/bblogger.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/bblogger.c b/src/bblogger.c
index c882275..ba3787e 100644
--- a/src/bblogger.c
+++ b/src/bblogger.c
@@ -146,6 +146,7 @@ static void parse_xorg_output(char * string){
     if (strstr(string, "Failed to load module \"kbd\"") ||
             strstr(string, "No input driver matching") ||
             strstr(string, "systemd-logind: failed to get session:") ||
+            strstr(string, "failed to set DRM interface version 1.4:") ||
             strstr(string, "Server terminated successfully")) {
       /* non-fatal errors */
       prio = LOG_DEBUG;
